src/store/modules/people.js

```diff
@@ -340,7 +340,9 @@ const actions = {
   },
 
   async fetchPersonTasks({ commit, state, rootGetters }, personId) {
-    const tasks = await peopleApi.getPersonTasks(personId)
+    let tasks = await peopleApi.getPersonTasks(personId)
+    const doneTasks = await peopleApi.getPersonDoneTasks(personId)
+    tasks = tasks.concat(doneTasks)
```

Comment:
You can call these 2 endpoints in parallel with a Promise.all() for perf purposes.